### PR TITLE
Adds CTAs to design guides and updates header

### PR DESCRIPTION
**Pull request by mnaumanali94**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit ecd4f01ce8a05c1eb2b84d852084dffe2b7ba2d5

https://deploy-preview-1613--stoplightio.netlify.app

File: netlify/subpages/api-documentation-guide.md [hunk 1]
```diff
@@ -10,6 +10,17 @@ publishedDate: 2019-02-11T17:37:38.030Z
 author: ''
 title: API Documentation & API Description Document Guide
 subtitle: Learn all there is about API Documentation through our comprehensive guide
+hero:
+  ctas:
+      - color: blue2
+        href: https://stoplight.io/welcome/create
+        large: true
+        title: Create API Documentation
```
Comment (paulatulis) [inline netlify/subpages/api-documentation-guide.md:18]: This is 'Generate API Documentation' in the other two tabs. Does that matter?

File: netlify/subpages/api-types-soap-vs-rest.md [hunk 2]
```diff
@@ -12,6 +12,17 @@ publishedDate:
...
 subtitle: >-
   SOAP APIs vs REST APIs - exploring the differences and similarities between these two approach to web services.
+hero:
+  ctas:
+      - color: blue2
+        href: https://stoplight.io/welcome/create
+        large: true
+        title: Design & Document REST APIs
+      - color: blue2
+        href: 'https://stoplight.io/'
```
Comment (paulatulis) [inline netlify/subpages/api-types-soap-vs-rest.md:22]: Is this the correct link?

File: netlify/subpages/api-types.md [hunk 3]
```diff
@@ -13,6 +13,17 @@ title: Types of APIs & Popular REST API Protocol
...
   Discussing different types of APIs, alongside protocols and standards, such as Open APIs, Internal APIs, Partner APIs,
   Composite APIs, RESTFUL, JSON-RPC, XML-RPC, and SOAP.
+hero:
+  ctas:
+      - color: blue2
+        href: https://stoplight.io/welcome/create
+        large: true
+        title: Design & Document with Stoplight
+      - color: blue2
+        href: 'https://stoplight.io/'
```
Comment (paulatulis) [inline netlify/subpages/api-types.md:23]: Is this the correct link?

File: netlify/subpages/microservices-guide.md [hunk 4]
```diff
@@ -7,6 +7,30 @@ relatedTags:
...
 title: Microservices Guide
 subtitle: Characteristics of microservices, best practices of microservice architecture, and tips for developers.
+hero:
+  ctas:
+      - color: blue2
+        href: https://stoplight.io/welcome/create
+        large: true
+        title: Design & Document with Stoplight
+      - color: blue2
+        href: 'https://stoplight.io/'
```
Comment (paulatulis) [inline netlify/subpages/microservices-guide.md:17]: Is this the correct link?

**Review by paulatulis — CHANGES_REQUESTED** [review]

**Comment by paulatulis** [comment]
Looks good to me! Called out a few things that I am not sure about, but you know better than me most of the time, so if they are correct, you can just disregard. 

Also, I like that these are called out, but there's a lot of whitespace. The two open source tiles have more going on, so it's not as bare. Is there a way we can set it up similarly? What do you think? 
Screen Shot 2020-09-29 at 12 53 36 PM

**Review by paulatulis — COMMENTED** [review]
All of the CTAs look good to me! Still curious what your thoughts are on the new dropdown menu whitespace. I think it looks a bit awkward.

**Comment by mnaumanali94** [comment]
> All of the CTAs look good to me! Still curious what your thoughts are on the new dropdown menu whitespace. I think it looks a bit awkward.

Good catch. Reduced the padding.

**Review by paulatulis — APPROVED** [review]

**Pull request merged**